src/runtime/components/navigation/Stepper.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <ol :class="[ui.base]">
+    <li
+      v-for="(item, index) of items"
+      :key="index"
+      ref="itemRefs"
+      :class="[ui.list.base]"
+    >
+      <div
+        v-if="displayDivider"
+        :class="[index > 0 ? 'visible' : 'invisible', ui.list.step.divider]"
+      />
+      <UButton
+        color="white"
+        variant="ghost"
+        :disabled="item.disabled"
+        :class="[
+          ui.list.step.base,
+          ui.list.step.background,
+          ui.list.step.padding,
+          ui.list.step.size,
+          ui.list.step.font,
+          selectedIndex === index ? ui.list.step.active : ui.list.step.inactive
+        ]"
+        @click="onChange(index)"
+      >
+        <div :class="[ui.list.step.leading.wrapper]">
+          <UIcon
+            v-if="(selectedIndex > index && completedIcon)"
+            :name="completedIcon"
+            :class="[ui.list.step.leading.icon]"
+          />
+          <UIcon
+            v-else-if="(selectedIndex === index && activeIcon)"
+            :name="activeIcon"
+            :class="[ui.list.step.leading.icon]"
+          />
+          <UIcon
+            v-else-if="(selectedIndex < index && pendingIcon)"
+            :name="pendingIcon"
+            :class="[ui.list.step.leading.icon]"
+          />
+          <UAvatar
+            v-else
+            :alt="(index + 1).toString()"
+            :size="ui.list.step.leading.size"
+          />
+        </div>
+        {{ item?.label }}
+      </UButton>
+      <div
+        v-if="displayDivider"
+        :class="[index + 1 < items.length ? 'visible' : 'invisible', ui.list.step.divider]"
+      />
+    </li>
+  </ol>
+</template>
+
+<script lang="ts">
+import type { StepItem } from '../../types/stepper'
```

Comment:
You're missing some `vue` imports here. There is no error, but this won't work when using the module otherwise.

```ts
import { computed, defineComponent } from 'vue'
```

Reply:
You're also missing the imports of the `Icon`, `Avatar`, `Button` components. Here is an example: https://github.com/nuxt/ui/blob/dev/src/runtime/components/navigation/VerticalNavigation.vue#L58

Reply:
Fixed